fvp.mk

```diff
@@ -126,6 +181,11 @@ linux-cleaner: linux-cleaner-common
 # OP-TEE
 ################################################################################
 OPTEE_OS_COMMON_FLAGS += CFG_ARM_GICV3=y
+
+ifeq ($(USE_MEASURED_BOOT),1)
+	OPTEE_OS_COMMON_FLAGS += CFG_DT=y CFG_CORE_TPM_EVENT_LOG=y CFG_CORE_ASLR=n
```

Comment:
Why `CFG_CORE_ASLR=n`?

Reply:
No particular reason. I guess that option was on the original patch I based mine on and I just left it there. It shouldn't cause any problem to have it removed (in fact, it is not set in the original fvp.mk) so I will remove it.

I will wait a bit before I push that change in case I have more review comments in the meantime.